cmd/view-fits/main.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2015 The astrogo Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"image"
+	"image/color"
+	"image/draw"
+	"log"
+	"os"
+
+	"github.com/astrogo/fitsio"
+
+	"golang.org/x/exp/shiny/driver"
+	"golang.org/x/exp/shiny/screen"
+	"golang.org/x/mobile/event/key"
+	"golang.org/x/mobile/event/paint"
+)
+
+var (
+	hduArray []fitsio.Image
+)
+
+func init() {
+
+	var inputFileName string
+	var inputFile *os.File
+	var fitsFile *fitsio.File
+	var err error
+
+	if len(os.Args) < 2 {
+		log.Fatal("no image file specified")
+	}
+
+	// Getting the file argument.
+	inputFileName = os.Args[1]
```

Comment:
so... what about the cases where one provides multiple filenames?
couldn't they also be seamlessly handled?
ie:
```sh
$> view-fits file0.fits file1.fits
```

at the very least, they shouldn't be dropped silently on the floor.